spender/data/__init__.py

```diff
@@ -0,0 +1 @@
+from .sdss import BOSS, SDSS
```

Comment:
There's a reason I left this one out. The data handling of different survey will/might require special packages, so I don't want to force everyone to install those just to get one survey to load. Please remove this one.

Reply:
removed, and updated README.md example